File: presentation/Priklady/c/priklm00.c
#include <stdio.h>

int vypocet(int n) {
	int v =1;
	while (n > 0) {
		v = v * (n % 10);
		n = n / 10;
		printf("n=%d,v=%d\n",n,v);
	}
	printf("=============\n");
	return v;
}

int main(){
	int cislo;
	printf("\n Zadat cislo (n): ");
	scanf("%d",&cislo);
	printf("\n");

	printf("Vysledok: %d\n",vypocet((float)cislo));
	
	return(0);
}

File: presentation/Priklady/ocaml/priklm00.ml
let rec funkcia n v = 
  if n = 0 then
    v
  else
    funkcia (n/10) (v * (n mod 10)) ;;

Printf.printf "Zadat cislo: %!";;

let n = read_int ()
in Printf.printf "Zadane je %d a vysledok je %d\n%!" n (funkcia n 1);;
